.obelisk/impl/github.json

```diff
@@ -1,8 +1,8 @@
 {
   "owner": "obsidiansystems",
   "repo": "obelisk",
-  "branch": "dylang/update-nix",
```

Comment:
Had to switch branches due to this being on top of master instead of develop which introduced the 8.10 support